torch/distributions/transformed_distribution.py

```diff
@@ -48,19 +50,21 @@ class TransformedDistribution(Distribution):
     :class:`~torch.distributions.relaxed_categorical.RelaxedOneHotCategorical`
     """
 
-    arg_constraints: dict[str, constraints.Constraint] = {}
+    arg_constraints: dict[str, Constraint] = {}
```

Comment:
not a classvar right?

Reply:
Do we need to type these as typing.ClassVar?

Reply:
`wishart.py` also has the line:

```python
self.arg_constraints["df"] = constraints.greater_than(event_shape[-1] - 1)
```

which would not be allowed if `arg_constraints` were a `ClassVar`. At the same time, this line appears in the tests:

```python
values = [
    self._perturb_tensor(
        value, Dist.arg_constraints.get(key, constraints.real)
    )
    for key, value in zip(keys, values)
]
```

which does assume that it is a `ClassVar`.